scripts/init_chain.sh

```diff
@@ -56,6 +56,14 @@ set_host_genesis() {
     jq "del(.app_state.interchain_accounts)" $genesis_config > json.tmp && mv json.tmp $genesis_config
     interchain_accts=$(cat $SCRIPT_DIR/config/ica.json)
     jq ".app_state += $interchain_accts" $genesis_config > json.tmp && mv json.tmp $genesis_config
+
+    # Optionally harshen slashing parameters
+    if [[ "$HARSHEN_SLASHING" == "true" ]]; then
+        sed -i -E 's|"signed_blocks_window": "100"|"signed_blocks_window": "10"|g' $genesis_config
+        sed -i -E 's|"min_signed_per_window": ""0.500000000000000000"|"min_signed_per_window": ""1.000000000000000000"|g' $genesis_config
```

Comment:
1 seems a bit too harsh (will we get any accidental slashes + jailing if a container gets too slow?), should we do 0.85 instead? Requires them to be down for 2 blocks which is still quite fast!

Reply:
Good point, I can make it a little more conservative! I also just realized our blocks are 1s. 

However, I do think we should either keep it strict and keep it optional, or relax it a tad and enable these by default. I'm not really sure how often slow containers cause missed blocks. 

Maybe we leave the min_signed_per_window at 0.5 and do signed_blocks_window as either 10 or 20?
Then I think they have to miss 5 or 10 blocks which is probably both unlikely to occur by accident and fast enough when it occurs intentionall?